extras/indexingExample/src/main/java/RyaDirectExample.java

```diff
@@ -789,7 +789,7 @@ private static void createPCJ(final Configuration conf)
 			for(Map.Entry<Key,Value> entry: scanner) {
 				System.out.println("PCJ entry :" + entry.getKey() );
 			}
-			
+			scanner.close();
```

Comment:
This needs to happen in a finally block to ensure it'll be closed even if an exception is thrown.